Apps/W1/HybridGP/app/src/codeunits/GPCloudMigration.codeunit.al

```diff
@@ -111,16 +107,13 @@ codeunit 4025 "GP Cloud Migration"
             HelperFunctions.UpdateGlobalDimensionNo();
         end;
 
-        CreateDataMigrationStatusRecords(Database::"G/L Account", GPAccount.Count(), 4090, 4017);
-        CreateDataMigrationStatusRecords(Database::"Customer", GPCustomer.Count(), 4093, 4018);
-        CreateDataMigrationStatusRecords(Database::"Vendor", GPVendor.Count(), 4096, 4022);
-        CreateDataMigrationStatusRecords(Database::"Item", GPItem.Count(), 4095, 4019);
+        CreateAllDataMigrationStatusRecords(DataMigrationEntity);
```

Comment:
the 'All' in the name infers we are creating all records when we are not. Suggest removing 'All' from the procedure name.

Reply:
Good point. I'm renaming to CreateConfiguredDataMigrationStatusRecords since that better describes what it's for.